frontend/src/features/user-settings/user-settings.tsx

```diff
@@ -0,0 +1,165 @@
+import {
+  Divider,
+  Stack,
+  Text,
+  UnstyledButton,
+  Group,
+  rem,
+  Title,
+  Flex,
+  Breadcrumbs,
+  Anchor,
+} from '@mantine/core'
+import { IconChevronRight, IconArrowLeft } from '@tabler/icons-react'
+import { VisibilitySettings } from '../visibility-settings'
+import { useState } from 'react'
+import { useAppSelector } from '../../redux'
+
+function UserSettings() {
```

Comment:
NItpick: Use `Settings` for the meantime, we're inclined to have these settings shared across the proposal as first iteration.